packages/govern-console/src/containers/CreateDao/CreateDaoProgress.tsx

```diff
@@ -248,7 +248,7 @@ const CreateDaoProgress: React.FC<{
     <ProgressComponent
       image={progressImage}
       title={'Creating your DAO'}
-      subTitle={'Hold tight your transaction is under process'}
+      subTitle={'Your transaction is in progress.'}
       progressList={progressList}
       info={'Please be patient and do not close this window until it finishes.'}
       action={action}
```

Comment:
I see the info `Please be patient...` even after the DAO is created.. is this intentional?

Reply:
missed another change for `Register token on Aragon voice → Registering token on Aragon Voice.`

Reply:
first comment: it was not in the scope of this task, I asked team leader and it is now is include.
secon comment: done